yarn-project/p2p/src/services/peer-manager/peer_manager.ts

```diff
@@ -41,8 +41,12 @@ export class PeerManager {
   private heartbeatCounter: number = 0;
   private displayPeerCountsPeerHeartbeat: number = 0;
   private timedOutPeers: Map<string, TimedOutPeer> = new Map();
-  private trustedPeers: Set<PeerId> = new Set();
+  private trustedPeers: Set<string> = new Set();
   private trustedPeersInitialized: boolean = false;
+  private privatePeers: Set<string> = new Set();
+  private privatePeersInitialized: boolean = false;
+
+  private protectedPeersNumber: number = 0;
```

Comment:
Rather than a protectedPeers number we can enforce that every private peer is part of the trusted peer set? then the length number will be the protected peers Number?